### ISSUE TITLE
missing generics for struct `ratatui::Frame`

### ISSUE THREAD

**Issue by ericcarmi** [ratatui-org/ratatui#569]
## Description
I cannot run example code in a new project. Running hello_world and other examples work when I clone the repo, but when adding to a new project, I am seeing `error[E0107]: missing generics for struct `ratatui::Frame``

## To Reproduce
cargo new tui-test
cargo add ratatui crossterm
copy code from hello_world into main.rs

## Expected behavior
expecting it to work as it did in the example

## Screenshots
<img width="731" alt="error" src="https://github.com/ratatui-org/ratatui/assets/25715335/a72b775c-1b23-4aa4-9a82-804fc45cf9b0">

## Environment

- OS: MacOS Ventura 13.4.1
- Terminal Emulator: Alacritty
- Font:
- Crate version: 0.23.0
- Backend:

feels like this is a really basic error and I am missing something that was not explained in docs, but I could not find anything in existing issues

**Comment by a-kenji** [comment]
> expecting it to work as it did in the example

Did you try using an example from the version that you are using?

**Comment by ericcarmi** [comment]
> > expecting it to work as it did in the example
> 
> Did you try using an example from the version that you are using?

I cloned the repo and ran the hello_world example. The tag on the repo says 0.23.1-alpha.6, and I am on 0.23.0 -- does that difference matter?

**Comment by a-kenji** [comment]
Yes, the the difference has to matter. Otherwise there could not be work in progress in this repo.

This is the `v0.23.0` tag https://github.com/ratatui-org/ratatui/releases/tag/v0.23.0 
you can also check the tag out in the clone.

**Comment by a-kenji** [comment]
>I cloned v0.23.0, the example runs just fine. Any other ideas?

Maybe I am misunderstanding you, but isn't your Problem solved then?
Now you have a working example if it runs just fine, or?

**Comment by ericcarmi** [comment]
> > I cloned v0.23.0, the example runs just fine. Any other ideas?
> 
> Maybe I am misunderstanding you, but isn't your Problem solved then? Now you have a working example if it runs just fine, or?

Sorry, the example ran from the repo. I thought it wasn't running in the other project but I forgot to copy the newly updated example. The example does run in my test project with 0.23.0. Thanks for the help

**Comment by joshka** [comment]
We recently changed Frame to no longer need a generic parameter. It's not yet released to a major version but is live in the alpha versions that are released every week.

See https://github.com/ratatui-org/ratatui/blob/main/BREAKING-CHANGES.md#generic-backend-parameter-removed-from-frame-530

**Comment by joshka** [comment]
Unpinning this as v0.24.0 is now released.
